tests/test_continue_simulation.py

```diff
@@ -25,124 +25,149 @@
 
 
 class TestContinueSimulation:
-    def get_oldest_time(self, ts):
-        return max(
-            [
-                ts.node(ts.first().roots[i]).time
-                for i in range(0, len(ts.first().roots), 1)
-            ]
-        )
+    def verify(self, ts, new_ts, time, continue_nodes=None):
+        assert round(new_ts.max_root_time - time, 10) == round(ts.max_root_time, 10)
+        assert ts.num_mutations == new_ts.num_mutations
+        assert ts.num_sites == new_ts.num_sites
+        assert list(ts.tables.sites.position) == list(new_ts.tables.sites.position)
+        if continue_nodes is None:
+            continue_nodes = ts.samples()
+        section = list(set(continue_nodes).intersection(new_ts.samples()))
+        assert len(section) == 0
+        for tree in new_ts.trees():
+            assert new_ts.node(tree.root).time > time
+        # check that subsetting to all nodes present before time gets back orig. tables
```

Comment:
I'd say first mark `continue_nodes` as samples, and then to
```
sub_tables = new_ts.simplify(continue_nodes).tables
sub_tables.provenances.clear()
orig_tables = ts.simplify(continue_nodes).tables
orig_tables.provenances.clear()
assert sub_tables == orig_tables
```
There can be tricky sorting issues, but I think this should work.